### ISSUE TITLE
Can't eat Blue Pill

### ISSUE THREAD

**Issue by WarpCircuit** [simeonradivoev/MatterOverdrive#151]
There's nothing in the logs about it either.

**Comment by shadowfacts** [comment]
What do you mean? Does the game crash when you try to eat it? Does the game send you back to the main menu when you eat? Can you not eat **because your hunger bar is full**?

**Comment by WarpCircuit** [comment]
Ok sorry about that. I made that post earlier to see if anyone had a similar issue with a solution.

The issue is that instead of the player consuming the pill in the instance of a right click, nothing happens. Between this time, I've found out that it's separate mod that's causing this issue and I should be back to you in a day or two as to which ones causing this, but at this point in time I don't have any idea who the culprit is. Until I can isolate the mod that's causing this, this is really irrelevant as the Mad Scientist still works to become an android. Thanks for making such an awesome mod anyways :+1:  . Once again, sorry about that.

tl;dr Some other mod funks up the Matter Overdrive blue pill. At this point in time, I don't have a culprit.

**Comment by shadowfacts** [comment]
No problem

**Comment by WarpCircuit** [comment]
Funny story. I finally had some time to test this and realized I was using the wrong pill. Whoops.

I'm so sorry. You can close this issue. It was simply the result of me and my team being complete ignorant imbeciles. I'm sorry to have wasted your time >:
